securedrop/client.py

```diff
@@ -37,31 +41,86 @@ def make_dict(self):
         }
 
 
+class AESWrapper(object):
+
+    def __init__(self, key):
+        self.bs = AES.block_size
+        self.key = hashlib.sha256(key.encode()).digest()
+
+    def encrypt(self, raw):
+        verify = hashlib.sha256((raw.encode())).hexdigest()
+        raw = self._pad(raw)
+        iv = Crypto.Random.get_random_bytes(AES.block_size)
```

Comment:
is there a benefit to manually passing in an IV? if not, we can remove this

Reply:
I suppose there is not, we are assuming there won't be tons of contacts, or enough for someone to form an attack by finding repeated messages.

I will leave this for now though.